docs/framework/install/guide-for-developers.md

```diff
@@ -33,7 +33,7 @@ manager: "wpickett"
 The fact that later .NET Framework 4.x versions are in-place updates to earlier versions means that you *cannot* install an earlier version listed in the table if a later version is already installed. For example, you cannot install the .NET Framework 4.6 on a Windows 10 November Update system, since the .NET Framework 4.6.1 is preinstalled on it.    
   
 > [!NOTE]
->  For information on the .NET Framework 3.5, see [Installing the .NET Framework 3.5 on Windows 8 and later versions](../Topic/Installing%20the%20.NET%20Framework%203.5%20on%20Windows%208,%20Windows%208.1%20and%20Windows%2010.md).  
+>  For information on the .NET Framework 3.5, see [Installing the .NET Framework 3.5 on Windows 8 and later versions](~/docs/framework/install/dotnet-35-windows-10.md).  
```

Comment:
The link you've provided is a user installation topic, not a developer installation topic. The correct link is [Installing the .NET Framework 3.5 on Windows 8, Windows 8.1 and Windows 10](net-framework-3-5-on-windows-8-plus.md).